components/event-feed-service/pkg/integration_test/feed_counts_test.go

```diff
@@ -194,6 +195,672 @@ func TestFeedCountsCountOnlyFilteredUsers(t *testing.T) {
 	runCases(t, cases)
 }
 
+func TestFeedCountsTypeCountsProjectFilter(t *testing.T) {
```

Comment:
The tests are mostly from https://github.com/chef/automate/blob/099a38e60c4617ca0d54fc358bc46b11fcca8653/components/config-mgmt-service/integration_test/event_counts_test.go#L123